core/src/main/scala/core/builder/api_json/upgrades/RemoveApiDocElement.scala

```diff
@@ -0,0 +1,14 @@
+package builder.api_json.upgrades
+
+import play.api.libs.json.JsObject
+
+/**
+ * In Oct 2022 we removed support for the top level 'apidoc' node which allowed
+ * the user to specify the version of API Builder. This feature was never
+ * implemented.
+ */
+object RemoveApiDocElement extends Upgrader {
+  def apply(js: JsObject): JsObject = {
+    js - "apidoc"
```

Comment:
why remove this field in code? why not reject the upload?

Reply:
experimenting with auto upgrade for a change to interfaces... feels annoying to require developers to change their spec (don't want to introduce a breaking change here for devs)